homeassistant/components/mqtt/device_trigger.py

```diff
@@ -290,7 +290,7 @@ async def async_removed_from_device(hass: HomeAssistant, device_id: str) -> None
 
 async def async_get_triggers(
     hass: HomeAssistant, device_id: str
-) -> list[dict[str, Any]]:
+) -> list[dict[str, str]]:
```

Comment:
I need to double-check this

Reply:
There's no need to have this strict limitation. However, the items in the returned dict must be JSON serializable.

Reply:
I have adjusted #72126 accordingly.
Most integrations only return strings in the dict, so I have kept both possibilities in the pylint plugin.

Reply:
OK. Can we make the expected type for these functions visible as a type def or `Protocol` somehow?